01-filedir.md

```diff
@@ -73,29 +91,26 @@ $ pwd
 /Users/nelle
 ~~~
 
-> ## Home directory {.callout}
+> ## Home Directory Variation {.callout}
 >
 > The home directory path will look different on different operating systems.
 > On Linux it will look like `/home/nelle`,
```

Comment:
This is determined by the HPC IT staff. For example I have:

`/usit/abel/u1/username`

This is basically "abbreviation of the IT staff departments name"/"name of the university's HPC cluster"/"some disk name"/username.